src/components/atoms/dropdown/modal.tsx

```diff
@@ -230,13 +240,18 @@ const styles = StyleSheet.create({
     borderBottomColor: colors.dot
   },
   searchInput: {
-    ...text.default,
+    ...text.default
   },
   searchUnderlined: {
     textDecorationLine: 'underline'
   },
   icon: {
     paddingHorizontal: 16
+  },
+  noPlaceholder: {
+    minWidth: 100,
+    borderBottomWidth: 1,
+    borderBottomColor: colors.black
```

Comment:
If we have to turn off the placeholder on Android with screen reader on, so that the accessiblityLabel is read, we need the input to be visible and big enough to press